package.json

```
@@ -42,7 +42,7 @@
   "peerDependencies": {
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "sweetalert2": "^9.7.0"
+    "sweetalert2": "^10.0.0"
```

I think this PR will be non-breaking (semver-minor) if we just make this change right?
```suggestion
    "sweetalert2": "^9.7.0 || ^10.0.0"
```

---

This would allow us to make more minor and patch releases that developers can use without being forced to upgrade to sweetalert2 v10

---

Done 👍